x-pack/plugins/security_solution/public/detections/pages/detection_engine/chart_panels/index.tsx

```diff
@@ -0,0 +1,206 @@
+        <FullHeightFlexItem grow={2}>
+          {isLoadingIndexPattern ? (
+            <EuiLoadingSpinner data-test-subj="trendLoadingSpinner" size="xl" />
+          ) : (
+            <AlertsHistogramPanel
+              alignHeader="flexStart"
+              chartHeight={TRENT_CHART_HEIGHT}
+              chartOptionsContextMenu={chartOptionsContextMenu}
+              defaultStackByOption={trendChartStackBy}
+              filters={alertsHistogramDefaultFilters}
+              onFieldSelected={updateCommonStackBy0}
+              panelHeight={TREND_CHART_PANEL_HEIGHT}
+              query={query}
+              showCountsInLegend={true}
+              showGroupByPlaceholder={true}
+              showTotalAlertsCount={false}
+              stackByLabel={GROUP_BY_LABEL}
+              title={title}
+              titleSize={'s'}
+              signalIndexName={signalIndexName}
+              updateDateRange={updateDateRangeCallback}
+              runtimeMappings={runtimeMappings}
+            />
+          )}
+        </FullHeightFlexItem>
+      )}
+
+      {alertViewSelection === 'table' && (
+        <AlertsCountPanelFlexItem grow={1}>
+          {isLoadingIndexPattern ? (
+            <EuiLoadingSpinner data-test-subj="tableLoadingSpinner" size="xl" />
+          ) : (
+            <AlertsCountPanel
+              alignHeader="flexStart"
+              chartOptionsContextMenu={chartOptionsContextMenu}
+              filters={alertsHistogramDefaultFilters}
+              panelHeight={TABLE_PANEL_HEIGHT}
+              query={query}
+              runtimeMappings={runtimeMappings}
+              setStackByField0={updateCommonStackBy0}
+              setStackByField1={updateCommonStackBy1}
+              signalIndexName={signalIndexName}
+              stackByField0={countTableStackBy0}
+              stackByField1={countTableStackBy1}
+              title={title}
+            />
+          )}
+        </AlertsCountPanelFlexItem>
+      )}
+
+      {alertViewSelection === 'treemap' && (
+        <FullHeightFlexItem grow={1}>
+          {isLoadingIndexPattern ? (
+            <EuiLoadingSpinner data-test-subj="treemapLoadingSpinner" size="xl" />
+          ) : (
+            <AlertsTreemapPanel
+              addFilter={addFilter}
+              alignHeader="flexStart"
+              chartOptionsContextMenu={chartOptionsContextMenu}
+              isPanelExpanded={isTreemapPanelExpanded}
+              filters={alertsHistogramDefaultFilters}
+              query={query}
+              setIsPanelExpanded={setIsTreemapPanelExpanded}
+              setStackByField0={updateCommonStackBy0}
+              setStackByField1={updateCommonStackBy1}
+              signalIndexName={signalIndexName}
+              stackByField0={riskChartStackBy0}
+              stackByField1={riskChartStackBy1}
+              title={title}
+              riskSubAggregationField="signal.rule.risk_score" // TODO: 'kibana.alert.rule.risk_score' returns no values
```

Comment:
is this still todo?

Reply:
@andrew-goldstein I believe it's `kibana.alert.risk_score` but I would still confirm that